pangeo-binder/values.yaml

```diff
@@ -189,19 +193,39 @@ kube-lego:
 
 dask-gateway:
   gateway:
-    clusterManager:
-      image:
-        name: daskgateway/dask-gateway-server
-        tag: 0.7.0
-        pullPolicy: IfNotPresent
-      clusterStartTimeout: 600
-      workerStartTimeout: 600
+    prefix: "/services/dask-gateway"
+    image:
+      name: daskgateway/dask-gateway-server
+      tag: 0.7.1
+      pullPolicy: IfNotPresent
+
+    backend:
+      scheduler:
+        cores:
+          request: 0.8
+          limit: 1
+        memory:
+          request: 1G
+          limit: 2G
+        extraPodConfig:
+#          name: scheduler-{JUPYTERHUB_USER}-{uuid}
+          tolerations:
+            - key: "k8s.dask.org/dedicated"
+              operator: "Equal"
+              value: "scheduler"
+              effect: "NoSchedule"
+            - key: "k8s.dask.org_dedicated"
+              operator: "Equal"
+              value: "scheduler"
+              effect: "NoSchedule"
+
       worker:
         extraContainerConfig:
           securityContext:
             runAsGroup: 1000
             runAsUser: 1000
         extraPodConfig:
+#          name: worker-{JUPYTERHUB_USER}-{uuid}
```

Comment:
I commented this line taken from our dask-kubernetes config b/c I didn't get a chance to try it, but right now `kubectl get pods -A` doesn't include any info about the user:
```
prod           dask-gateway-d2psk                                       1/1     Running   0          10m
prod           dask-gateway-hcvjt                                       1/1     Running   0          11m
```